clients/js/src/ChromaClient.ts

```diff
@@ -50,6 +63,15 @@ export class ChromaClient {
             this.api = new DefaultApi(apiConfig);
         }
 
+        // TODO: include this? we have this in python
```

Comment:
In python, this is used to validate the requested tenant/database is created. We should do the same here.

Reply:
we cant run an `async` function in a `constructor`. In the `embedding_functions` we get around this by calling our async function when we run `generate`. We don't want to do that here because it would lead to a lot of overhead on every request.... what if we just don't check?

Reply:
Can't we synchronously call the api?

Reply:
any ideas here?

https://github.com/chroma-core/chroma/pull/1398/files#diff-d568e2646599d1674863ba07a10fb3de7a3776f64b7d44c8fe9d8dcb62a64126R74-R79

Reply:
What version of chroma is the server?